src/registrar/assets/sass/_theme/_buttons.scss

```diff
@@ -211,6 +211,18 @@ a.usa-button--unstyled:visited {
     align-items: center;
 }
 
+.usa-button--justify-right {
+  justify-content: right;
+}
+
+
+.dotgov-table a,
```

Comment:
Looks like an artifact of some merge I did?

Reply:
This did come in via a merge-conflict resolution

Reply:
Blocking: Plz remove :)